### PR TITLE
Add missing plugin TypoScript

### PR DESCRIPTION
**Pull request by franzholz**

### CODE REVIEW

File: Documentation/Exceptions/1540246570.rst [hunk 1]
```diff
@@ -6,14 +6,63 @@ TYPO3 Exception 1540246570
...
+    \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPlugin(
+
+However the tt_content TypoScript for the list type is not added if you look into the Active TypoScript
+in the backend. 
+The extension TypoScript is like this:
+
+..  code-block:: typoscript
+
+    plugin.tx_myextension {
+      controller.action = USER_INT
```
Comment (linawolf) [inline Documentation/Exceptions/1540246570.rst:40]: You should state which TYPO3 version you are talking about. Starting with TYPO3 12 there is EXTBASEPLUGIN https://docs.typo3.org/m/typo3/reference-typoscript/main/en-us/ContentObjects/Extbaseplugin/Index.html...

File: Documentation/Exceptions/1540246570.rst [hunk 2]
```diff
@@ -6,14 +6,63 @@ TYPO3 Exception 1540246570
...
 
 --------------
 
 Occures if you pass the wrong pluginType to
 TYPO3\CMS\Extbase\Utility\ExtensionUtility::configurePlugin(...)
 
+No Content Object definition found at TypoScript object path "tt_content.list.20
+================================================================================
+
+If your extension is not activated, then you need to activate it.
```
Comment (froemken) [inline Documentation/Exceptions/1540246570.rst:24]: This is only valid for Standalone instances

File: Documentation/Exceptions/1540246570.rst (hunk 1 above)
Comment (franzholz) [reply Documentation/Exceptions/1540246570.rst:40]: It is for a general plugin.

File: Documentation/Exceptions/1540246570.rst (hunk 2 above)
Comment (franzholz) [reply Documentation/Exceptions/1540246570.rst]: What is a standalone instance?

Comment (franzholz) [reply Documentation/Exceptions/1540246570.rst]: This is called [Legacy Installation](https://docs.typo3.org/m/typo3/tutorial-getting-started/main/en-us/Installation/LegacyInstallation.html)

Comment (franzholz) [reply Documentation/Exceptions/1540246570.rst]: No, this error does not depend on TYPO3, but only on the extension.

File: Documentation/Exceptions/1540246570.rst [hunk 3]
```diff
@@ -6,14 +6,63 @@ TYPO3 Exception 1540246570
...
+    \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPlugin(
+
+However the tt_content TypoScript for the list type is not added if you look into the Active TypoScript
+in the backend. 
+The extension TypoScript is like this:
+
+..  code-block:: typoscript
+
+    plugin.tx_myextension {
+      controller.action = USER_INT
```
Comment (linawolf) [inline Documentation/Exceptions/1540246570.rst:40]:
```suggestion
      controller.action = EXTBASEPLUGIN
```

File: Documentation/Exceptions/1540246570.rst [hunk 4]
```diff
@@ -6,14 +6,63 @@ TYPO3 Exception 1540246570
...
+..  code-block:: typoscript
+
+    plugin.tx_myextension {
+      controller.action = EXTBASEPLUGIN
+
+
+Solution
+--------
+
+Always add the TypoScript for the tt_content list rendering.
```
Comment (linawolf) [inline Documentation/Exceptions/1540246570.rst:46]: the code below does not add TypoScript?

**Review by linawolf — CHANGES_REQUESTED** [review]
Please correct the code example

File: Documentation/Exceptions/1540246570.rst [hunk 5]
```diff
@@ -6,14 +6,63 @@ TYPO3 Exception 1540246570
...
 
 --------------
 
 Occures if you pass the wrong pluginType to
 TYPO3\CMS\Extbase\Utility\ExtensionUtility::configurePlugin(...)
 
+TYPO3 12 - General Plugin: No Content Object definition found at TypoScript object path "tt_content.list.20
+============================================================================================================
+
+If your extension is a general plugin and is not activated, then you need to activate it.
```
Comment (linawolf) [inline Documentation/Exceptions/1540246570.rst:24]:
```suggestion
If your extension is a general plugin, check if it is properly installed (and activated for legacy installations)
and the required TypoScript included.
```

File: Documentation/Exceptions/1540246570.rst (hunk 4 above)
Comment (franzholz) [reply Documentation/Exceptions/1540246570.rst:47]: addPlugin modifies $GLOBALS['TCA']['tt_content'] only. 

Only ExtensionManagementUtility::addPItoST43 does it.
Or simply use the code below.  I just mention one solution.

**Review by linawolf — APPROVED** [review]

File: Documentation/Exceptions/1540246570.rst (hunk 4 above)
Comment (linawolf) [reply Documentation/Exceptions/1540246570.rst:47]: I will make a follow up

**Pull request merged**